src/parsing.jl

```diff
@@ -100,8 +100,10 @@ is not in UTF-8.
 """
 bytes_view(pqv::PQValue) = unsafe_wrap(Vector{UInt8}, data_pointer(pqv), num_bytes(pqv) + 1)
 
+# bytes_view(pqv::PQValue) = bswap(unsafe_load(data_pointer(pq_value_bin)))
+
 Base.String(pqv::PQValue) = unsafe_string(pqv)
-Base.parse(::Type{String}, pqv::PQValue) = unsafe_string(pqv)
+Base.parse(::Type{String}, pqv::PQValue) where R = unsafe_string(pqv)
```

Comment:
```suggestion
Base.parse(::Type{String}, pqv::PQValue) = unsafe_string(pqv)
```

I'm guessing this was accidentally left in